### ISSUE TITLE
backend/risk_alert.py: Risk score double-counts negative_belief_rate and deep_negative_rate

### ISSUE THREAD

**Issue by wuxixixi** [wuxixixi/ProjectInsight#1026]
## Problem

In `get_risk_summary()` (lines 345-351), the risk score formula double-counts the negative belief dimension:

```python
risk_score = (
    negative_rate * t.weight_negative +           # 25%
    deep_negative_rate * t.weight_deep_negative +  # 25%
    polarization * t.weight_polarization +          # 25%
    silence_rate * t.weight_silence +               # 15%
    (1 - truth_rate) * t.weight_truth_deficit       # 10%
)
```

`negative_belief_rate` and `deep_negative_rate` are highly correlated (deep_negative is a subset of negative). With 50% combined weight, this inflates the risk score whenever negative beliefs are present.

## Example

If `negative_rate=0.5` and `deep_negative_rate=0.3`:
- Negative contribution: 0.5 * 0.25 + 0.3 * 0.25 = 0.2
- This is 50% of the `score_medium` threshold (0.25) from just the negative dimension alone

## Also: debunk_effect metric is never populated

Line 194 defines a rule for `debunk_effect` metric, but this value is never computed in `_get_current_state_dict()` (prediction.py). The rule will never trigger because `current_state.get('debunk_effect', 0)` always returns 0.

## Recommendation

1. Reduce negative dimension weights or use orthogonal metrics
2. Add `debunk_effect` computation to `_get_current_state_dict()`:
```python
# Compute debunk effectiveness after response is published
if engine.responded and len(engine.history) >= 2:
    current_truth = current.positive_belief_rate
    prev_truth = engine.history[-2].get('positive_belief_rate', 0)
    state_dict['debunk_effect'] = current_truth - prev_truth
```

## Severity
**Medium** - Risk scores may be misleading, one rule is dead code
